openstack/db/v1/instances/testing/requests_test.go

```diff
@@ -132,3 +133,31 @@ func TestResizeVolume(t *testing.T) {
 	res := instances.ResizeVolume(fake.ServiceClient(), instanceID, 4)
 	th.AssertNoErr(t, res.Err)
 }
+
+func TestAttachConfigGroup(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	req := `{
+    	"instance": {
+        	"configuration": "00000000-0000-0000-0000-000000000000"
+    	}
+	}`
+	fixture.SetupHandler(t, resURL, "PUT", req, "", 202)
```

Comment:
It looks like you're creating a "raw" HTTP request here. What we want to do is create an actual Gophercloud request. See [here](https://github.com/gophercloud/gophercloud/blob/master/openstack/db/v1/flavors/testing/requests_test.go#L88-L107) for an example. The other unit tests are all created in the same way, too, so you can browse the various `testing/requests_test.go` files for more examples. Let me know if you need help, though.